src/index.html

```diff
@@ -3,12 +3,27 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap&subset=cyrillic" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header >
+    <a href="#"><img class="logo" src="images/logo.png" alt="logo"></a>
+    <nav class="header-nav">
+      <ul class="list">
+        <li class="item"><a class="link is-active" href="#">Apple</a></li>
+        <li class="item"><a class="link" href="#">Samsung</a></li>
+        <li class="item"><a class="link" href="#">Смартфоны и телефоны</a></li>
+        <li class="item"><a class="link" href="#" data-qa="hover">Ноутбуки и компьютеры</a></li>
+        <li class="item"><a class="link" href="#">Гаджеты</a></li>
+        <li class="item"><a class="link" href="#">Планшеты</a></li>
+        <li class="item"><a class="link" href="#">Фото</a></li>
+        <li class="item"><a class="link" href="#">Видео</a></li>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.